reader/processor/processor.go

```diff
@@ -64,6 +70,13 @@ func ProcessFeedEntries(store *storage.Storage, feed *model.Feed) {
 		entry.Content = sanitizer.Sanitize(entry.URL, entry.Content)
 
 		entry.ReadingTime = calculateReadingTime(entry.Content)
+
+		if config.Opts.FetchYouTubeWatchTime() {
+			if matches := youtubeRegex.FindStringSubmatch(entry.URL); len(matches) == 2 {
+				entry.ReadingTime = fetchYouTubeWatchTime(entry.URL)
```

Comment:
- Calculating twice the reading time should be avoided (see line 72). If the entry is a YouTube video, the normal reading time will be calculated and then, the Youtube's website will be scrapped just after.
- If the function `fetchYoutubeWatchTime` fails, then it should probably fallback to the default behaviour.
- How do you handle video with a duration of `PT0M0S`? it seems to be used for Live stream or Radio.